scripts/install_build_tools.sh

```diff
@@ -4,24 +4,35 @@ set -e
 . /etc/lsb-release
 echo "Distribution code name : ${DISTRIB_CODENAME}"
 apt-get update
-if [ "$DISTRIB_CODENAME" = "bionic" ];then 
-    echo "Installing bionic build tools ..."
-    DEPENDENCIES=(
-        build-essential 
-        cmake 
-        googletest 
-        google-mock
-        gdb
-        git)
-else
-    echo "Installing other linux distribution (focal or jammy) build tools ..."
-    DEPENDENCIES=(
+apt-get -y install 
+DEPENDENCIES=(
+        wget
+        ca-certificates
         build-essential
-        cmake
-        build-essential
-        libgtest-dev
-        libgmock-dev
         gdb
         git)
-fi
-DEBIAN_FRONTEND=noninteractive apt install --no-install-recommends --yes --quiet "${DEPENDENCIES[@]}"
\ No newline at end of file
+DEBIAN_FRONTEND=noninteractive apt install --no-install-recommends --yes --quiet "${DEPENDENCIES[@]}"
+# Installing CMake 
```

Comment:
We should probably do all this in `/tmp` to avoid colliding with anything in the current directory.